src/Button.js

```diff
@@ -51,6 +51,10 @@ class Button extends React.Component {
         {...this.props}
         href={href || '#'}
         className={classNames(this.props.className, classes)}
+        tabIndex="0"
+        role="button"
+        ariaLabel={this.props.ariaLabel}
+        onKeyUp={this.handleBtnKeyUp}
```

Comment:
few things here. SafeAnchor already handles adding the button role, and tabIndex (its job is to ensure that "link" buttons are accessible). `onKeyUp` is also attached to a handler that doesn't exist so I'm not sure what the goal with that is.